nservicebus/sagas/index.md

```diff
@@ -22,6 +22,24 @@ Design processes with more than one remote call to use sagas.
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
 
+## No saga instance available
+
+If a message that doesn't start a saga is received but no related saga instance is found, then by default that message is discarded, i.e. it is possible that a saga instance had been already completed when further messages arrive and such messages should be discarded.
+
+NOTE: Always assume that messages can be delivered out of order, i.e. due to invoking the error recovery mechanism, network latency, and/or concurrent message processing.
+
+A similar message discarding behavior can also be invoked in case of saga instance creation. When a message is received that expects a saga instance to be there but that message that would create the saga instance has not yet been received or completed processing.
+
+There are two options to avoid discarding incoming messages in this scenario:
+
+- Let the saga instance be created for the message processing that previously assumed the saga instance would already exist using `IAmStartedBy<T>`
```

Comment:
I feel it'd be much simpler to explain using an example, e.g. let's take saga from some sample we have, that is started by a message X, and then updated when a message Y arrives... Otherwise it'll be hard to follow.